### ISSUE TITLE
Interaction with nav causes loss of contrast

### ISSUE THREAD

**Issue by aaaaargZombies** [Kcaden20/critical-code-cookbook#43]
## Issue I'm experiencing

On all pages.

Hover and Focus states for some elements in the navigation set the `color` and `outline` properties to `--bgColor`.

- When combined `--colorWhite: #f5f5f5` & `--colorYell: #FFF799;` have a contrast ratio of 1.01
- When combined `--colorWhite: #f5f5f5` & `--colorYell: #FFF799;` have a contrast ratio of 2.05
- When combined `--colorWhite: #f5f5f5` & `--colorPurp: #CC66FF;` have a contrast ratio of 2.79

## Steps to reproduce

Visit [criticalcode.recipes/about](https://criticalcode.recipes/about) and hover mouse over navigation links in header.

## Potential fixes

- Remove hover effect.
- Create a "highlight" effect by altering `background-color` instead of `color` so as to contrast against black text.
- Choose new colors that contrast well with the header background.
